### ISSUE TITLE
translating conj.value

### ISSUE THREAD

**Issue by krisgesling** [lb803/list-manager-skill#3]
Hi Luca,

Just wanted to check for people who are translating this to Skill to other languages.

`conj.value` contains 
> "conj,and"

I'm presuming only the "and" should be translated in other languages eg Indonesian:
> "conj,dan"

**Comment by lb803** [comment]
Yes, precisely, that was my intention. Have I followed bad practice there?

**Comment by krisgesling** [comment]
It may be the right solution for your use case, however it might be sufficient to rename it `and.dialog` and use:
```python
self.translate("and")
```
We also have a new method available [`join_list()`](https://github.com/MycroftAI/lingua-franca/blob/master/lingua_franca/format.py#L465) that would replace your string method nicely
```Python
from mycroft.util.format import join_list

join_list(items, "and", lang=self.lang)
```
We saw many Skills all doing the same thing so made it a little bit easier :)

Edit: but looking back at my original question - seems obvious to me now that conj should not get translated.

**Comment by lb803** [comment]
Thank you for this! This is a very valuable feedback!
